indigo_app/templates/place/works.html

```diff
@@ -104,7 +104,13 @@ <h3 class="mb-0">{{ latest_completeness_stat.n_complete_works }} of {{ latest_co
                     {% endif %}
                   </div>
 
-                  <div class="col-2">
+                  {% if work.metrics %}
+                    <div class="col-1">
```

Comment:
It's better to always include the column (ie. move the `if` inside the `div`), otherwise in the case where there are no metrics, that row will have the incorrect number of columns and it won't line up with the other rows.